packages/calcite-components/src/components/accordion-item/accordion-item.scss

```diff
@@ -1,210 +1,204 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-accordion-border-color: Specifies the component's border color.
```

Comment:
We can discuss this separately, but maybe we should consider lifting all parent/child props to the parent or list them separately. With our current process, `accordion` would show matching CSS props and `accordion-item` would  have a mix in their respective doc pages. cc @geospatialem @DitwanP